app/uk/gov/hmrc/apisubscriptionfields/connector/Model.scala

```diff
@@ -27,3 +27,7 @@ private[connector] case class SubscriberRequest(callBackUrl: String, subscriberT
 private[connector] case class UpdateSubscriberRequest(subscriber: SubscriberRequest)
 
 private[connector] case class UpdateSubscriberResponse(boxId: BoxId)
+
+private[connector] case class UpdateCallBackUrlRequest(callBackUrl: String)
+
+private[connector] case class UpdateCallBackUrlResponse(successful: Boolean, errorMessage: Option[String])
```

Comment:
Not sure about this - surely the response success or error can be determine by http status code and therefore a flag to say if it worked or not is not needed?

Reply:
We need to return the error to the UI.

Reply:
Its actually the main point of the changes we have introduced